assignment-client/src/octree/OctreeServer.cpp

```diff
@@ -1386,6 +1391,11 @@ QString OctreeServer::getFileLoadTime() {
     return result;
 }
 
+double OctreeServer::getFileLoadTimeSeconds()
+{
+     return getLoadElapsedTime() / 1000000.0;
```

Comment:
what is the 1000000.0? seems like a magic number. If its MICRO_SECONDS_IN_SECOND perhaps define a constant with that name